src/bors/mod.rs

```diff
@@ -26,6 +29,68 @@ pub enum CheckSuiteStatus {
     Success,
 }
 
+#[derive(Clone, Debug, PartialEq)]
+pub enum PullRequestStatus {
```

Comment:
We decided to model the PR state and the mergeability status separately in https://github.com/rust-lang/bors/pull/219. So this should only be Open/Draft/Closed/Merged.